products/error_tracking/frontend/components/ExceptionAttributesPreview/ExceptionAttributesPreview.tsx

```diff
@@ -66,7 +66,7 @@ export function PropertyWrapper({
         return <></>
     }
     return (
-        <LemonTag>
+        <LemonTag className="bg-white">
```

Comment:
This will be white in dark mode afaik. I think it's `bg-light` or something we need